server/helpers/verifyEmailMessage.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @description verifyEmailMessage contains the email message template
+ * @param {string} token - The token generated for the user
+ * @returns {object} - contains the mail message template
+ */
+
+const verifyEmailMessage = (token) => {
+  if (token === undefined || token === '') {
+    const err = new Error('token is invalid');
+    err.status = 404;
```

Comment:
Not sure this should be a 404 error

Reply:
You are right. It should be 401 error (Unauthorised)

Reply:
Feedback implemented. I had changed the 404 error to 401 error.